Documentation/SwiftBundler.docc/configuration.md

```diff
@@ -21,8 +21,12 @@ icon = "icon.png"
 prebuild_script = "./utils/prebuild.sh"
 postbuild_script = "./utils/postbuild.sh"
 
-[apps.HelloWorld.extra_plist_entries]
+[apps.HelloWorld.plist]
 commit = "{COMMIT_HASH}" # This could be any key-value pair, 'commit' is just an example
+CFBundleURLTypes = [ { type = "dict", value = { CFBundleTypeRole = 'Viewer', CFBundleURLName = 'HelloWorld', CFBundleURLSchemes = [ 'helloworld' ] } } ]
+# You can also define many other kinds of platform-specific fields, as an example, above is
```

Comment:
I think `platform-specific fields` is not really what this is demonstrating, more that you can define any complex fields that you want (whether they're platform-specific or not).